### DISCUSSION TITLE
Error on click for Update to 1.2.2

### DISCUSSION THREAD

**Discussion by morphinestyle** [akeeba/panopticon#792] [Q&A (Community Support)]
This update to 1.2.2 didn't work as usual, it returns this error:

```
Incomplete installation
It looks like you are trying to use Akeeba Panopticon from the Git repository, however you have not completed all build steps.
Please refer to the README.md file on the project's repository for building Akeeba Panopticon for use.

```

![pano8](https://github.com/user-attachments/assets/5c0f23f9-dd0a-4d45-9669-a5f2ac128e1d)

How can I solve it?
Do I unzip version 1.2.2 again and overwrite the old installation?

**Comment by nikosdion** [comment]
The message means that the `vendor` folder is missing or incomplete.

Retry extracting the update ZIP file.

If this doesn't work, ask your host to reset the PHP OPcache. Ideally, you should have OPcache expire at 2 to 5 seconds, and make sure that `opcache_invalidate` is not disabled on your server.

**Reply by morphinestyle** [reply]
Same as here again: https://github.com/akeeba/panopticon/issues/754

**Reply by nikosdion** [reply]
Fabio, this is a misconfiguration of your server. If you don't fix it, it will keep happening. I already told you what to do. 

> you should have OPcache expire at 2 to 5 seconds

If you still don't get it, here's what you need to configure _at the very least_ in PHP:
```
opcache.validate_timestamps=1
opcache.revalidate_freq=2
```

Please do [read the fine manual](https://www.php.net/manual/en/opcache.configuration.php#ini.opcache.validate-timestamps).

**Reply by morphinestyle** [reply]
![Screenshot_2024-08-27-23-53-36-85_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/12758dee-539c-4c1a-9fe4-e71c697e0bcf)

Strange situation, I'll try to understand more. Thank you!
